### ISSUE TITLE
Non-deterministic results when using prepare_su_noise_multires

### ISSUE THREAD

**Issue by Miczu** [bvhari/ComfyUI_SUNoise#4]
Images generated with prepare_su_noise_multires don't appear to be deterministic.

I suspect interpolate as the source of issue.
`
noise = torch.nn.functional.interpolate(noise, size=latent_image.size()[1:], mode='nearest-exact')
`

https://pytorch.org/docs/stable/generated/torch.nn.functional.interpolate.html
`This operation may produce nondeterministic gradients when given tensors on a CUDA device. See [Reproducibility](https://pytorch.org/docs/stable/notes/randomness.html) for more information.`

I tried to replace the method with something else, but I failed :(

**Comment by Miczu** [comment]
Maybe the issue is more complex interaction between comfy nodes. Not I get deterministic results with both noise types, but I changed my experimental values in other nodes xd

**Comment by bvhari** [comment]
It has been pretty much deterministic for me in use, so if you can reproduce the issue consistently, I'll investigate this further.

**Comment by Miczu** [comment]
Thanks for reply, I think I have better grasp on what the core issue is.

So I started comfy ui from scratch. With fixed seed I queued: Seed A, Seed A, Seed A+1, Seed A, Seed A, Seed A, Seed A.
With deterministic result I expected to have 2 distinct images, but I got 4 distinct images:
1) Seed A - Image # 1
2) Seed A - Image # 2
3) Seed A+1 - Image # 3
4) Seed A - Image # 4
5) Seed A - Image # 2
6) Seed A - Image # 2
7) Seed A - Image # 2

Simplest scenario to reproduce would be to generate for A, A+1, A - 1st and 3rd generation should be the same, but aren't for me.

Based on above, it feels like there is some leftover state being mixed in and it is stable if the same seed is used in consecutive runs.
I don't see this problem when I use "RandomNoise" node from ComfyUI.

If I run the same seed x3 times, but 2nd run has different lora (1st and 3rd run have the same lora). 
The 1st & 3rd run generate the same image as expected (2nd is different). So model stuff doesn't bleed between runs. for SUNoise.

**Comment by bvhari** [comment]
Thanks for the details, let me try to reproduce the issue from my side.
